packages/common/src/runtimes/NodeRuntime.ts

```diff
@@ -163,14 +163,18 @@ class NodeRuntimeFileSystem implements RuntimeFileSystem {
   }
 
   glob(patterns: ReadonlyArray<string>) {
-    return fastGlob(patterns as string[], {
+    return glob({
+      patterns: patterns as string[],
```

Comment:
using `patterns` inside options is deprecated, although it's unlikely for a major tinyglobby version to happen anytime soon you should use it as first argument instead

Reply:
thanks. updated!